initScripts/x86_64__Ubuntu_16.04__Docker_17.06.sh

```diff
@@ -8,7 +8,8 @@ readonly DOCKER_VERSION="17.06.0"
 readonly SWAP_FILE_PATH="/root/.__sh_swap__"
 export docker_restart=false
 
-export BASE_DIR="$SHIPPABLE_DIR/$(cat /proc/sys/kernel/random/uuid)"
+export BASE_UUID="$(cat /proc/sys/kernel/random/uuid)"
```

Comment:
This is a small change in the way the container is named. It's currently called `shippable-exec-nodeId`. Since the node can support multiple reqProcs, this will now be `reqProc-UUID`.

Let me know if I should open a different issue for this.